core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -256,7 +307,29 @@ get_database_sort(Db1, Db2) ->
 
 -spec refresh(kz_term:ne_binary()) -> 'ok'.
 refresh(Database) ->
-    _Pid = spawn('kapi_maintenance', 'refresh_views', [Database]),
+    refresh_views(Database, kz_datamgr:db_classification(Database)).
+
+-spec refresh_views(kz_term:ne_binary(), kz_datamgr:db_classification() | kz_term:ne_binary()) -> 'ok'.
+refresh_views(_Database, 'undefined') ->
+    'ok';
+refresh_views(Database, 'modb') ->
+    kazoo_modb:refresh_views(Database),
+    _ = kazoo_bindings:map(binding({'refresh', Database}), [Database]),
+    'ok';
+refresh_views(Database, 'account') ->
+    refresh_account_db(Database);
+refresh_views(Database, _) ->
+    _ = kz_datamgr:refresh_views(Database),
+    _ = kazoo_bindings:map(binding({'refresh', Database}), [Database]),
+    'ok'.
```

Comment:
why not handle this in a simpler way ? call `kz_datamgr:refresh_views`  and then `kazoo_bindings:map(binding({'refresh', Database}), [Database]),` ?

Reply:
I don't understand, this is doing what you told.

Reply:
you handle it differently for `modb` and  `account`

Reply:
to be clear, it is ok to handle `account` and `modb` differently in `refresh` but `refresh_views` should be simple as the last `function` clause (i was trying to show that but the lines showed only these)

Reply:
because account have to maps refresh_account binding, previously there was no map for modb binding

Reply:
but you are calling the binding for `refresh_account` from refresh_views and it should be called from `refresh`.
also, in refresh_views we're calling the binding for `refresh` and it should be `refresh_views`, and `refresh` binding should be called from `refresh`